src/static/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
What is this used for? Can that be removed?

Reply:
The manifest is used to provide information about the website (https://developer.mozilla.org/en-US/docs/Web/Manifest). Blockstack uses the site name, icon,.. during authentication.